eitprocessing/datahandling/sequence.py

```diff
@@ -16,7 +16,7 @@
 
 
 @dataclass(eq=False)
-class Sequence(Equivalence, SelectByTime):
+class Sequence(Equivalence, SelectByIndex):
```

Comment:
see my comment in continuousdata

Reply:
Fixed in 77604884aeba19b0c1844a3ffc7d3a391a6ac750.